include/IECore/IECore.h

```diff
@@ -44,13 +44,19 @@
 namespace IECore
 {
 
+/// Returns the milestone version for the IECore library
```

Comment:
Note I haven't added any `#defines` or `MAKE_COMPATIBILITY_VERSION` equivalent like we have in `Gaffer/Version.h`... should I? Up till now it seems we've relied on the SConstruct adding those defines for internal files, and external projects have been left to their own devices for compile-time conditionals...

Reply:
I think it'd be worthwhile, and now would be a natural time to do it.

Reply:
Added in 4a0b449